Example/Example Swift/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,27 @@
+/* 
+  Localizable.strings
+  GiniCapture
+
+  Created by Peter Pult on 16/06/16.
+  Copyright © 2016 Gini. All rights reserved.
```

Comment:
This header is out of date. Change it please to your name and the current date (or last Friday's date). You can also change 2016 to 2021.

Reply:
Updated ;)